Added documentation how to post the SDLDidUpdateProjectionView notification properly (#467)
smartdevicelink/sdl_app_library_guides: docs/Video Streaming for Navigation Apps/Supporting Haptic Input/index.md

```
~|

> SDL can only automatically detect `UIButton`s and anything else that responds `true` to `canBecomeFocused`. This means that custom `UIView` objects will *not* be found. You must send these objects manually, see "Manual Focusable Rects".
> 
> !!! NOTE
> In case the renderingType is `SDLCarWindowRenderingTypeLayer`, then the `SDLDidUpdateProjectionView` notification should only be sent in the overridden `viewDidLayoutSubviews` method of your `rootViewController` or else the views might not be rendered as expected.
> 

Reviewer: ```suggestion
!!! NOTE
When your renderingType is `SDLCarWindowRenderingTypeLayer`, the `SDLDidUpdateProjectionView` notification should only be sent in the overridden `viewDidLayoutSubviews` method of your `rootViewController`. If you do not, your haptic rects may not update as you expect.

SDL can only automatically detect `UIButton`s and anything else that responds `true` to `canBecomeFocused`. This means that custom `UIView` objects will *not* be found. You must send these objects manually, see "Manual Focusable Rects".

```
